### ISSUE TITLE
Setting the clear button height results in error messages due to it not being on the stage

### ISSUE THREAD

**Issue by ArtyomZorin** [kylecorry31/gnome-shell-extension-do-not-disturb#39]
When the extension is being enabled at startup or upon refreshing Gnome Shell the date menu popup wouldn't have appeared yet, so the Clear button is not on the stage. Because of this, the code to change the height of the clear button results in these error messages.

```
(gnome-shell:24861): St-CRITICAL **: 17:01:43.650: st_widget_get_theme_node called on the widget [0x55a0f7e5e050 StBoxLayout ("Do not disturb")] which is not in the stage.

(gnome-shell:24861): St-CRITICAL **: 17:01:43.650: st_widget_get_theme_node called on the widget [0x55a0f7e64420 StBoxLayout.popup-menu-item do-not-disturb:last-child first-child ("Do not disturb")] which is not in the stage.

(gnome-shell:24861): St-CRITICAL **: 17:01:43.650: st_widget_get_theme_node called on the widget [0x55a0f7e5e050 StBoxLayout ("Do not disturb")] which is not in the stage.

(gnome-shell:24861): St-CRITICAL **: 17:01:43.651: st_widget_get_theme_node called on the widget [0x55a0f7e64420 StBoxLayout.popup-menu-item do-not-disturb:last-child first-child ("Do not disturb")] which is not in the stage.

(gnome-shell:24861): St-CRITICAL **: 17:01:43.651: st_widget_get_theme_node called on the widget [0x55a0f7ff3ad0 StBin:first-child] which is not in the stage.

(gnome-shell:24861): St-CRITICAL **: 17:01:43.651: st_widget_get_theme_node called on the widget [0x55a0f7e5e050 StBoxLayout ("Do not disturb")] which is not in the stage.

(gnome-shell:24861): St-CRITICAL **: 17:01:43.651: st_widget_get_theme_node called on the widget [0x55a0f7e64420 StBoxLayout.popup-menu-item do-not-disturb:last-child first-child ("Do not disturb")] which is not in the stage.

(gnome-shell:24861): St-CRITICAL **: 17:01:43.651: st_widget_get_theme_node called on the widget [0x55a0f7ff3ad0 StBin:first-child] which is not in the stage.

(gnome-shell:24861): St-CRITICAL **: 17:01:43.651: st_widget_get_theme_node called on the widget [0x55a0f80510f0 StBin.toggle-switch toggle-switch-us:last-child first-child] which is not in the stage.

(gnome-shell:24861): St-CRITICAL **: 17:01:43.651: st_widget_get_theme_node called on the widget [0x55a0f7e5e050 StBoxLayout ("Do not disturb")] which is not in the stage.

(gnome-shell:24861): St-CRITICAL **: 17:01:43.651: st_widget_get_theme_node called on the widget [0x55a0f7e64420 StBoxLayout.popup-menu-item do-not-disturb:last-child first-child ("Do not disturb")] which is not in the stage.

(gnome-shell:24861): St-CRITICAL **: 17:01:43.651: st_widget_get_theme_node called on the widget [0x55a0f7ec5240 StLabel:last-child ("Do not disturb")] which is not in the stage.
```

These messages appear when running "gnome-shell --replace" in the Terminal. I am using Gnome Shell 3.30

**Comment by kylecorry31** [comment]
@ArtyomZorin I saw that you made a few commits to a fork of this, feel free to open a PR and I will review it if you would like to contribute back to this extension

**Comment by ArtyomZorin** [comment]
Thanks for the heads up! It looks like I forgot to PR these commits, but they should be available to review and merge now in PR #41
